src/sentry/runner/commands/cleanup.py

```diff
@@ -57,47 +57,44 @@ def cleanup(days, project, concurrency, silent, model, router):
     from django.db import router as db_router
     from sentry.app import nodestore
     from sentry.db.deletion import BulkDeleteQuery
-    from sentry.models import (
-        ApiGrant, ApiToken, Event, EventMapping, Group, GroupRuleStatus,
-        GroupTagValue, LostPasswordHash, TagValue, GroupEmailThread, FileBlob,
-        EventTag,
-    )
+    from sentry import models
```

Comment:
all of these changes are unrelated - i was going to add GroupSnooze here but it was more complex than anticipated